evm/evm-processor/src/mapping/schema.ts

```diff
@@ -64,6 +74,7 @@ export function getTxProps(fields: FieldSelection['transaction'], forArchive: bo
             s: withSentinel('Transaction.s', '0x', BYTES),
             yParity: option(natural),
             chainId: option(natural),
+            authorizationList: option(array(Authorization)),
```

Comment:
@eldargab do we need ability to select each field manually or it's fine and we can load the whole array of structures?

Reply:
Let's expose it as a whole.